apps/server/src/shared/repo/account/account.repo.integration.spec.ts

```diff
@@ -82,10 +84,19 @@ describe('account repo', () => {
 		});
 	});
 
+	describe('findByUserId', () => {
+		it('should findByUserId', async () => {
+			const accountToFind = accountFactory.build();
+			await em.persistAndFlush(accountToFind);
+			em.clear();
+			const account = await repo.findByUserId(accountToFind.user.id);
+			expect(account.id).toEqual(accountToFind.id);
+		});
+	});
 	describe('findOneByUser', () => {
```

Comment:
The test cases in line 105 'should throw an error' and 109 'should not respond with an account for wrong id given' seem to be duplicates. Also the comment in line 105 is unclear